options/train/BasicVSR/train_BasicVSR_GAN_C64N15L5_201600kft_pix1e-2per1GAN5e-3_B4G4.yml

```diff
@@ -0,0 +1,142 @@
+# general settings
+name: 1001_ReGAN_C64N15L5_201600kft_pix1e-2per1GAN5e-3_B4G4
+model_type: VideoRecurrentGANModel
+scale: 4
+num_gpu: 8  # set num_gpu: 0 for cpu mode
+manual_seed: 0
+
+# dataset and data loader settings
+datasets:
+  train:
+    name: REDS
+    type: REDSRecurrentDataset
+    dataroot_gt: datasets/REDS/train_sharp_with_val.lmdb
+    dataroot_lq: datasets/REDS/train_sharp_bicubic_with_val.lmdb
+    dataroot_flow: ~
+    meta_info_file: basicsr/data/meta_info/meta_info_REDS_GT.txt
+    val_partition: REDS4  # set to 'official' when use the official validation partition
+    test_mode: False
+    io_backend:
+      type: lmdb
+
+    task: VSR
+    num_frame: 15
+    num_interpolate: 0
```

Comment:
do we have `num_interpolate: 0` in the current dataloder?

You need to compare the two configure files  (the given RecurrentGAN option and the BasicVSR option) to see their differences and update it.